test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -377,6 +377,38 @@ def test_emacs(image):
     run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+@pytest.mark.integration("awscli")
+def test_awscli(image):
+    """
+    Ensure that awscli is installed on every image
+
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    container_name = get_container_name("awscli", image)
+    start_container(container_name, image, ctx)
+
+    run_cmd_on_container(container_name, ctx, "which aws")
+    run_cmd_on_container(container_name, ctx, "aws --version")
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("boto3")
+def test_boto3(mxnet_inference):
+    """
+    Ensure that boto3 is installed on every image
```

Comment:
nit: we may not make boto3 a requirement going forward, so let's call out that we are installing on MXNet inference containers in the docstring. It looks like we have boto3 in recent tensorflow2 inference images as well, so you could make this 

```
def test_boto3(inference):
"""
Ensure that boto3 is installed in relevant inference images
"""
fw, _ = get_framework_and_version_from_tag(inference)

# Note: boto3 is currently only regularly installed on tf and mxnet inference images
if fw not in ("tensorflow", "mxnet"):
    pytest.skip(f"Skipping test on unsupported framework {fw}")
```

Reply:
@arjkesh @jeet4320 We need `boto3` and `botocore` to make the EC2 service calls in DLC telemetry scripts work.

Reply:
> @arjkesh @jeet4320 We need `boto3` and `botocore` to make the EC2 service calls in DLC telemetry scripts work.

@saimidu @jeet4320 - I don't think this is accurate. We do not need boto3, just botocore for the script. Botocore is a downstream dependency of awscli.

Reply:
awscli brings botocore, so we test awscli for all dlc images then